scipy/stats/_morestats.py

```diff
@@ -4419,13 +4415,6 @@ def circmean(samples, high=2*pi, low=0, axis=None, nan_policy='propagate'):
         High boundary for the sample range. Default is ``2*pi``.
     low : float or int, optional
         Low boundary for the sample range. Default is 0.
-    axis : int, optional
-        Axis along which means are computed. The default is to compute
-        the mean of the flattened array.
```

Comment:
Please bring back `axis` as we discussed. The decorator uses it when it can, and in those cases it's more efficient than looping over the slices.